api/src/main/kotlin/edu/wgu/osmt/richskill/RichSkillRepository.kt

```diff
@@ -285,12 +281,22 @@ class RichSkillRepositoryImpl @Autowired constructor(
             }?.let { jobsToRemove.addAll(it.filterNotNull()) }
         }
 
+        skillUpdate.authors?.let { lookupKeywords(it, KeywordTypeEnum.Author) }
         skillUpdate.keywords?.let { lookupKeywords(it, KeywordTypeEnum.Keyword) }
         skillUpdate.certifications?.let { lookupReferences(it, KeywordTypeEnum.Certification) }
         skillUpdate.standards?.let { lookupAlignments(it, KeywordTypeEnum.Standard) }
         skillUpdate.alignments?.let { lookupAlignments(it, KeywordTypeEnum.Alignment) }
         skillUpdate.employers?.let { lookupReferences(it, KeywordTypeEnum.Employer) }
 
+        val authorsUpdate = if (addingAuthors.size > 0 || removingAuthors.size > 0) {
```

Comment:
Where are we using this, looks like it should be used between line 319 and line 331, but I think the authors changes are included in the allKeyWordsUpdate object. So my guess is that this is not needed.

Reply:
This was removed.